README.md

```diff
@@ -25,13 +25,19 @@ On release, automated continuous integration tests run the pipeline on a full-si
 
 ## Pipeline summary
 
-<!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
-
-1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
-2. Map reads to reference ([`BWA mem`](http://bio-bwa.sourceforge.net/bwa.shtml), `BWA mem2`) with the option to ([`merge`](http://www.htslib.org/doc/samtools-merge.html))
-3. Mark duplicated reads ([`GATK4 MarkDuplicates`](https://gatk.broadinstitute.org/hc/en-us/articles/360037052812-MarkDuplicates-Picard-))
-4. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
-5. Variant calling ([deepvariant](https://github.com/google/deepvariant) and [glnexus](https://github.com/dnanexus-rnd/GLnexus))
+1. Metrics: [`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/), [`MultiQC`](http://multiqc.info/)
+2. Data preprocessing: [`bwamem2`](http://bio-bwa.sourceforge.net/bwa.shtml) (can [`merge`](http://www.htslib.org/doc/samtools-merge.html)), [`MarkDuplicates`](https://broadinstitute.github.io/picard/command-line-overview.html#MarkDuplicates),
+3. Variant calling + multiple calls are aggregated:
+    1. SNVs: [`DeepVariant`](https://github.com/google/deepvariant)
```

Comment:
Should the GATK option be mentioned?
And Sentieon alternative: [`DNAscope`](https://bioconda.github.io/recipes/sentieon/README.html) and [`DNAModelApply`](https://bioconda.github.io/recipes/sentieon/README.html)
Plus for the mitochondria: [Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/360037593851-Mutect2)
(And if we want to include the Sentieon version, it would be [TNscope](https://bioconda.github.io/recipes/sentieon/README.html)